src/server/appserver.cpp

```diff
@@ -1909,6 +1909,7 @@ void AppServer::onRequestReceived(int id, RequestNum num, const QByteArray &para
         }
         case RequestNum::UTILITY_DISPLAY_CLIENT_REPORT: {
             sentry::pTraces::basic::AppStart().stop();
+            break;
```

Comment:
It seems to be an unrelated fix. What actual issue was caused by the missing `break` statement?

Reply:
The next C/S request (SYNC_SETSUPPORTSVIRTUALFILES) is called, fortunately without parameters (so syncDbId=0).
=> Error in the log ""Error in setSupportsVirtualFiles for syncDbId=0"